src/plugins/slick.cellselectionmodel.ts

```diff
@@ -48,7 +48,7 @@ export class SlickCellSelectionModel {
     this._options = Utils.extend(true, {}, this._defaults, this._options);
     this._grid = grid;
     if (grid.hasDataView()) {
-      this._dataView = grid.getData<SlickDataView>();
+      this._dataView = grid.getData<CustomDataView>();
```

Comment:
we could probably use a type combo here, even though it probably won't make a difference but still would be more explicit `grid.getData<CustomDataView | SlickDataView>();`